### DISCUSSION TITLE
Space problem on host Disk (C:)

### DISCUSSION THREAD

**Discussion by Zillos** [Chia-Network/chia-blockchain#10722] [Support]
Hello everyone I'm new Here And I have a question about the space on disk C during plotting/farming. The Chia app stops sincying because the C: disk is full for no reason (i don't have C: as a plot folder)... my main c disk it's a 110 gb SSD. What's the problem? is there a solution or I'm doing something wrong?

**Comment by Gooseboy2234** [comment]
How big is you're current db folder?
Have you upgraded you're DB version yet?

**Reply by Zillos** [reply]
currently 80 gb... I work with 7 pc sharing the same wallet with tons of plot each, but have that one problem in every pc

what do you mean about upgrading DB version? I'm using the last version of Chia

**Reply by Gooseboy2234** [reply]
Chia version 1.3 came out with the ability to upgrade the database version from version 1 to 2, witch cuts down the size by about 50%

What OS are you on?

**Reply by Zillos** [reply]
I'm on Windows 10, just for farming because I have all the hard drives at full

**Reply by Gooseboy2234** [reply]
Do this, this should fix it https://m.youtube.com/watch?v=3HVcj4G4y_8

I upgraded a few days ago, for the 5/5 part there's 90,000 coins

**Reply by Gooseboy2234** [reply]
I tell you the coin amount as it can no joke take up to a day to upgrade on a modern system

**Reply by Gooseboy2234** [reply]
But you can't upgrade the database version right now, you first have to to delete two plots on Every machine that you need to create the database version on

**Reply by Zillos** [reply]
two plots on every machine or two plots on every hard drive??

**Reply by Gooseboy2234** [reply]
Two plots on every machine on the same hdd, and then perform a database Location transfer on to said hard drive. So you will actually have room to upgrade the database

**Reply by Gooseboy2234** [reply]
Do this https://spacefarmers.io/wiki/guides/farming/movedb then upgrade the database... delete all database V1 files and Leave V2 files. Then do https://spacefarmers.io/wiki/guides/farming/movedb again for c: drive

**Reply by Gooseboy2234** [reply]
Keep in mind though, the Db upgrade might only save you for a few months. And you might need to consider moving the db to one of you're hdd's on all of you're computers 24/7

**Reply by Zillos** [reply]
Thank you so much dude! you were a big help!! now's working well :D

**Reply by Gooseboy2234** [reply]
You're welcome! Good luck with you're adventures in the Chia project!

By the way, if the option shows up for you, don't forget to mark this thread as "issue solved."

**Reply by Gooseboy2234** [reply]
Btw, since your on windows, you can activate windows file compression on the DB folders to get an extra 5GB free!
